tests/performance/locust_files/metamodeling/passer.py

```diff
@@ -0,0 +1,15 @@
+import json
```

Comment:
Document a bit why we need these?

Reply:
OK, I will not do this further now. I think the main doc is in the locust file as well as the png I added to understand the setup